README.md

```diff
@@ -23,6 +23,11 @@ Presto has a comprehensive set of unit tests that can take several minutes to ru
 
     ./mvnw clean install -DskipTests
 
+
+## Presto native and Velox
+   Presto native is a C++ rewrite of presto worker. Presto native uses [Velox](https://github.com/facebookincubator/velox) as its primary engine to run presto workloads. [Velox](https://github.com/facebookincubator/velox) is a C++ database library which provides reusable, extensible, and high-performance data processing components. 
+
+
 ## Running Presto in your IDE
```

Comment:
It would be nice to provide instructions on how to build Presto Native . Maybe we could port the documentation you wrote some time back.

Reply:
There is documentation in the presto-native page : https://github.com/prestodb/presto/tree/master/presto-native-execution#building .

Reply:
Should we provide a link to that?